omero/developers/Modules/Api.txt

```diff
@@ -14,16 +14,16 @@ connection e.g. Python:
     adminService = session.getAdminService()                 # now we can get/create services
 
 -  The :javadoc:`Service factory API <slice2html/omero/api/ServiceFactory.html>`
-   has methods for creating Stateless and Stateful services (see below).
+   has methods for creating Stateless and Stateful services see below.
```

Comment:
The brackets were fine here, you'll need a comma after 'services' now you've removed it